core/views.py

```diff
@@ -40,6 +46,7 @@ def get_context_data(self: "PostDetailView", **kwargs: dict[str, Any]) -> dict[s
         context = super().get_context_data(**kwargs)
         comments = self.object.get_comments()
 
+        context["default_avatar_url"] = context["post"].author.avatar_url
```

Comment:
Please explain. Why you assign author avatar to default_avatar_url?

Reply:
at the previous stage it returned the image correctly

Reply:
I have already deleted it